lib/airtable.js

```diff
@@ -31,6 +31,9 @@ function Airtable(opts) {
         _noRetryIfRateLimited: {
             value: opts.noRetryIfRateLimited || Airtable.noRetryIfRateLimited || defaultConfig.noRetryIfRateLimited,
         },
+        _userTable: {
+            value: opts.userTable || Airtable.userTable
```

Comment:
@FranCogMents I wonder if we should be pulling this from some "metadata" endpoint from Airlock, instead of requiring a user to specify their user table on both client and server.

Reply:
@ethanlee16 good point! cc @arpanpal 

If we pull this data from Airlock this would incur an additional network request before doing executing the actual register/log-in request? If that's the case, I'd like us to avoid any network requests for the register/login to be successful.

In the short-run (this week), we could have the Airlock client specify the "user table name" and the Airlock server can parse that "user table name". This allows us to have the client be the single source of truth of where the "user table name" will be defined.

In the long-run -- (next week) we can have the Airlock client specify not only the table name, but also the column names of their "username" + "password" fields -- since currently on the Airlock server we need to specify the `username` table column name (we are assuming that `password` is called "password").  this long run sort of starts pushing some configuration on the client, which makes the Airlock Server more "transparent" and more responsive to real-time changes (Client decides to rename username field or password field vs having to go into the server and check the env variables + restart the server)

Reply:
> this would incur an additional network request before doing executing the actual register/log-in request? 

Yes, that's true. I imagine we can make it lightweight enough to not be noticeable, but we'd have to delay all other incoming requests until we know what that user table's name is.

> we could have the Airlock client specify the "user table name" and the Airlock server can parse that "user table name". 

I'm not sure I like the idea of the client having the source of truth. This would mean someone can craft a login/register request that looks up a non-user table that we may not want to expose any information about (including whether or not it exists).

Also: do we need to store the `_userTable` in the client at all? We don't need it for the actual login/register endpoints since the server is already configured to know what it is. Maybe I'm missing a potential use for this name though - open to thoughts!

Reply:
You're right @ethanlee16 , it's not necessary to have `_userTable` at all in the client right now, specifically for `login/register`since those endpoints are already predetermined by Airlock client/server.

I saw that _userTable was being used in the `AirlockContext.js` example
`Airlock.configure({
  apiKey: "airlock",
  endpointUrl: "http://localhost:4000",
  userTable: "Users"
});`

But besides this example, I think we're fine to remove this